models/nxm/ethereum/nxm_ethereum_schema.yml

```
@@ -0,0 +1,103 @@
+version: 2
+
+models:
+  - name: nxm_ethereum_quotation_trades
+    meta:
+      blockchain: ethereum
+      project: nxm
+      contributors: [zxf, jonah]
+    config:
+      tags: ['ethereum','nexusmutual','trades']
+    description: >
+        nexusmutual quotation trades on Ethereum
+    columns:
```

Hi @sharkxff , you can add an uniqueness test here.
```suggestion
    description: >
        nexusmutual quotation trades on Ethereum
    tests:
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - evt_block_number
            - evt_tx_hash
            - evt_index
    columns:
```